flexmeasures/ui/templates/crud/account.html

```diff
@@ -0,0 +1,155 @@
+{% extends "base.html" %} {% set active_page = "accounts" %}{% block title %} Account overview {% endblock %} {% block
+divs %} 
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-sm-2"></div>
+        <div class="col-sm-8">
+
+            <h1>Account overview</h1>
+            <div class="card">
+                <h3>Account</h3>
+
+                <table class="table table-striped">
+                    <tbody>
+                        <tr>
+                            <td>
+                                ID
+                            </td>
+                            <td>
+                                {{ account.id }}
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                Roles
+                            </td>
+                            <td>
+                                {{ account.account_roles | join(", ") }}
```

Comment:
This also shows only role IDs (see comment on accounts.html)

Reply:
Added the role names instead after PR625 was added.